app/views/users/show/pr/_icon.html.erb

```diff
@@ -1,3 +1,10 @@
+<% if valid_pr > 4 && pr.eligible?%>
```

Comment:
Should this not be moved down to be inside the `<% elsif pr.eligible? %>` block, so if its a bonus PR, we show the bonus icon, else we show the eligible icon?